02/app.js

```diff
@@ -1,2 +1,17 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+for (i = 0; i <= x; i++) {
```

Comment:
Uwaga na deklarację przed `i` tj. `let i = 0` - w taki sposób tworzymy zmienną lokalną (dostępną tylko wew. pętli), jesli nie napiszemy `let` to będzie to zmienna globalna (dostępna w całym programie)